packages/react-tc-ui-kit/src/components/Input/Input.js

```diff
@@ -1,29 +1,73 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useState } from 'react';
 
 import './Input.css';
 
 export default function Input({
-  style = {},
-  id,
+  style,
   className = '',
+  id,
   type = 'text',
-  value = '',
-  placeholder = '',
-  disabled,
-  onChange,
+  label,
+  value,
+  placeholder,
+  isInvalid = false,
+  validationMessages = [],
+  isMessagesAbsolute = false,
+  onChange = () => { },
   ...props
 }) {
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+
+  const invalidClassname = isInvalid ? 'tc-input--invalid' : '';
+  const inputPasswordClassname = type === 'password' ? 'tc-input__control--password' : '';
+  const errorsAbsoluteClassname = isMessagesAbsolute ? 'tc-input__errors--absolute' : '';
+
   return (
-    <input
-      style={style}
-      className={`tc-input ${className}`}
-      id={id}
-      type={type}
-      value={value}
-      placeholder={placeholder}
-      disabled={!!disabled}
-      onChange={onChange}
-      {...props}
-    />
+    <div className={`tc-input ${className} ${invalidClassname}`} style={style}>
+      {label && (
+        <label className="tc-input__label" htmlFor={id}>{label}</label>
+      )}
+
+      <div className="tc-input__box">
+        <input
+          id={id}
+          placeholder={placeholder}
+          className={`tc-input__control ${inputPasswordClassname}`}
+          type={isPasswordVisible ? 'text' : type}
+          value={value}
+          onChange={onChange}
+          {...props}
+        />
+
+        {type === 'password' && (
+          <button
+            className="tc-input__view-toggler"
+            type="button"
+            onClick={() => setIsPasswordVisible(!isPasswordVisible)}
+          >
+            {isPasswordVisible
+              ? (
+                <svg width="25" height="20" viewBox="0 0 25 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
i've just added proper svg resolving\
now you can import them as react components\
dont forget to add className and styles :)